### DISCUSSION TITLE
Set Rive animation to crash window

### DISCUSSION THREAD

**Discussion by 760900587** [community/community#47320] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

An exception occurred in removeView ， Please have a look

/art: art/runtime/java_vm_ext.cc:475] JNI DETECTED ERROR IN APPLICATION: JNI CallVoidMethodV called with pending exception java.lang.NoSuchMethodError: no non-static method

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @760900587 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
